tests/RazorLight.Tests/RazorLight.Tests.csproj

```diff
@@ -5,7 +5,15 @@
     <IsPackable>false</IsPackable>
     <PreserveCompilationContext>true</PreserveCompilationContext>
     <DefineConstants>$(DefineConstants);SOME_TEST_DEFINE</DefineConstants>
+  </PropertyGroup>
+  
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
```

Comment:
This checks for `netstandard2.0` but the target frameworks are `netcoreapp2.0;netcoreapp3.0` so this might not be necessary?

Reply:
That is a typo, I wanted to write `netcoreap2.0`. 
So I guess it just reverted back to `Default` and that indeed should work for both versions. 
So I can remove the LangVersion.